src/module-1/index.html

```diff
@@ -0,0 +1,162 @@
+    </div>
+  </div>
+
+  <div class="product">
+    <img class="product__img" src="img/products/jbl-speaker@2x.png" />
+    <div class="product__price">$199</div>
+    <p class="product__title">JBL Speaker</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
+          <span class="product__btn-text-1">WISHLIST</span>
+        </button>
+      </div>
+      <div class="product__btn">
+        <button class="product__btn-group-2">
+          <span class="product__btn--shopping-bag"></span>
+          <span class="product__btn-text-2">ADD TO CART</span>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="product">
+    <img class="product__img" src="img/products/iphone-x@2x.png" />
+    <div class="product__price">$899</div>
+    <p class="product__title">Apple iPhone X 128GB</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
+          <span class="product__btn-text-1">WISHLIST</span>
+        </button>
+      </div>
+      <div class="product__btn">
+        <button class="product__btn-group-2">
+          <span class="product__btn--shopping-bag"></span>
+          <span class="product__btn-text-2">ADD TO CART</span>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="product">
+    <img class="product__img" src="img/products/beats-headphones@2x.png" />
+    <div class="product__price">$459</div>
+    <p class="product__title">Beats Headphones</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
+          <span class="product__btn-text-1">WISHLIST</span>
+        </button>
+      </div>
+      <div class="product__btn">
+        <button class="product__btn-group-2">
+          <span class="product__btn--shopping-bag"></span>
+          <span class="product__btn-text-2">ADD TO CART</span>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="product">
+    <img class="product__img" src="img/products/macbook-pro@2x.png" />
+    <div class="product__price">$2499</div>
+    <p class="product__title">Apple Macbook Pro 512GB SSD</p>
+    <p class="product__description">
+      Redesigned from scratch and completely revised.
+    </p>
+    <div class="flex-row">
+      <div class="product__btn">
+        <button class="product__btn-group-1">
+          <span class="product__btn--heart"></span>
+          <span class="product__btn-text-1">WISHLIST</span>
```

Comment:
uppercase may be achieved with css `text-transform` as well, so you don't need to put text in caps